radial_kernel.hpp

```diff
@@ -175,12 +177,18 @@ class RadialDispersalKernel
     std::cauchy_distribution<double> cauchy_distribution;
     std::exponential_distribution<double> exponential_distribution;
     von_mises_distribution von_mises;
+    DeterministicDispersalKernel<IntegerRaster> deterministic_kernel;
+    bool deterministic_;
 public:
     RadialDispersalKernel(double ew_res, double ns_res,
                           DispersalKernelType dispersal_kernel,
-                          double distance_scale,
+                          double distance_scale = 1.0,
```

Comment:
Why add a default here? It should be a required parameter that we need from the user (or the calibration).

Reply:
I had the distance_scale and locator values set up so it would default to the standard cauchy/exponential distribution, but I can take it out.